src/lib/tradesHandler.ts

```diff
@@ -152,62 +153,11 @@ export const triggerClearOldBuyOrders = async () => {
   return { readyTrades };
 };
 
-const determineStopLossType = (
-  trade: ExtendedTradesDataType,
-  stopLossLimit: number,
-): TRADE_STATUS | undefined => {
-  const lastTradePrice = trade.lastTradePrice;
-  if (!lastTradePrice) return;
-
-  const movingAvg = trade.movingAvg10;
-
-  // Stop loss case (1)
-  if (trade.price - lastTradePrice >= stopLossLimit)
-    return TRADE_STATUS.STOP_LOSS_1;
-
-  if (!isToday(trade.created)) {
-    // Stop loss case (2)
-    if (lastTradePrice <= trade.price) return TRADE_STATUS.STOP_LOSS_2;
-
-    // Stop loss case (3) Take profit
-    if (movingAvg && lastTradePrice <= movingAvg)
-      return TRADE_STATUS.STOP_LOSS_3;
-  }
-
-  return;
-};
-
-/** After 10% increase in value, we take profit */
-const isTakePartialProfit = (trade: ExtendedTradesDataType) => {
-  if (trade.status === TRADE_STATUS.TAKE_PARTIAL_PROFIT) {
-    // We only want to do this once; Since it's already been done, the
-    // next sell should be a stop-loss to sell 100%
-    return false;
-  }
-
-  const lastTradePrice = trade.lastTradePrice;
-  return lastTradePrice && trade.price * 1.1 <= lastTradePrice;
-};
-
-const determineTradeStatus = (
-  trade: ExtendedTradesDataType,
-  stopLossLimit: number,
-): TRADE_STATUS | undefined => {
-  const stopLossType = determineStopLossType(trade, stopLossLimit);
-
-  if (stopLossType) {
-    return stopLossType;
-  } else if (isTakePartialProfit(trade)) {
-    return TRADE_STATUS.TAKE_PARTIAL_PROFIT;
-  }
-
-  return;
-};
-
 const depopulateTrade = (trade: ExtendedTradesDataType): TradesDataType => {
-  delete trade.lastTradePrice;
-  delete trade.movingAvg10;
-  return trade;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Is this a nice change if you have to disable eslint after the change? I don't know the benifit of doing the destruct instead of delete

Reply:
After I made the props required, I got this error:

![image](https://user-images.githubusercontent.com/1492711/214531118-5f8c3a78-1dfa-4b13-8f06-a5e97394420e.png)

And I don't want to modify a parameter object anyway. But apparently I can add this eslint rule instead: https://stackoverflow.com/a/69175834

Reply:
fixed as described above